### PR TITLE
MBS-10003: Generalize entity Aliases tabs to one file

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-10003

### CODE REVIEW

**Comment by reosarevok** [comment]
@yvanzo mentioned that artists have more content on their page - which is true. When we reach that point we could either have a separate template for that or add a conditional to the shared file.

File: root/entity/Aliases.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import chooseLayoutComponent from '../utility/chooseLayoutComponent';
+import AliasesComponent from '../components/Aliases';
```
Comment (yvanzo) [inline root/entity/Aliases.js]: `AliasesComponent` doesn’t exist

Comment (reosarevok) [reply root/entity/Aliases.js]: AliasesComponents is an alias for the default export of components/Aliases because this one is also Aliases.

Comment (mwiencek) [reply root/entity/Aliases.js]: Well, the only other way is to do `import {default as AliasesComponent}` but I'm not sure that's an improvement.

**Pull request merged**